DC-SLES-SAP-sol-automation

```diff
@@ -1,5 +1,5 @@
 MAIN="book_solution_based.xml"
-ROOTID="art-sol-automation"
+ROOTID="article-sol-automation"
```

Comment:
maybe we can leave out the 'sol' (for solution here) and call the root ID 'article-sap-automation' (which would also be closer to the title?)